From 0ddd152236a7f404076f8af8cfb5ce84c80c68cf Mon Sep 17 00:00:00 2001
From: Ilya Terentyev <bacondropped@gmail.com>
Date: Sun, 15 May 2016 21:23:28 +0300
Subject: [PATCH] install nokogiri outside of Makefiles

Nokogiri is a requirement of Capybara and builds native extensions on
installation, which may cause some problems with checkinstall. This
patch forces the installation of Nokogiri outside tools' Makefiles,
which fixes the issue.
---
 web/imgur-dl/Makefile  | 1 +
 web/myzuka-dl/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/web/imgur-dl/Makefile b/web/imgur-dl/Makefile
index 3ac2da4..c3df9ba 100644
--- a/web/imgur-dl/Makefile
+++ b/web/imgur-dl/Makefile
@@ -3,6 +3,7 @@
 all:
 
 install:
+	gem install nokogiri
 	bundle install
 	install imgur-dl /usr/local/bin
 	install imgur-dl.1 /usr/local/man/man1
diff --git a/web/myzuka-dl/Makefile b/web/myzuka-dl/Makefile
index a9f7a79..3c0a7df 100644
--- a/web/myzuka-dl/Makefile
+++ b/web/myzuka-dl/Makefile
@@ -3,5 +3,6 @@
 all:
 
 install:
+	gem install nokogiri
 	bundle install
 	install myzuka-dl /usr/local/bin
-- 
2.7.4

